.env

```
@@ -20,20 +20,34 @@ ITKDEV_TEMPLATE=symfony-6
 
 ###> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=09922bda5aedeb42a14d196a0955f790
+APP_SECRET=
 ###< symfony/framework-bundle ###
 
 ###> doctrine/doctrine-bundle ###
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
-DATABASE_URL="mysql://db:db@mariadb:3306/db?serverVersion=10.11.6-MariaDB&charset=utf8mb4"
-MOCK_DATABASE_URL="sqlite:///%kernel.project_dir%/var/mocks.sqlite"
+#
+# DATABASE_URL="sqlite:///%kernel.project_dir%/var/data_%kernel.environment%.db"
+# DATABASE_URL="mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8.0.32&charset=utf8mb4"
+# DATABASE_URL="mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=10.11.2-MariaDB&charset=utf8mb4"
+DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=16&charset=utf8"
 ###< doctrine/doctrine-bundle ###
 
 ###> symfony/mailer ###
-MAILER_DSN=smtp://mail:1025
+MAILER_DSN=null://null
```

This isn't really the used `MAILER_DSN`:

```suggestion
MAILER_DSN=smtp://mail:1025
```